release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1395,6 +1402,11 @@ In the table, features are marked with the following statuses:
 |
 |DEP
 
+|Support for Google Cloud Platform Workload Identity
+|-
+|-
+|xref:../release_notes/ocp-4-10-release-notes.adoc#ocp-4-10-removed-feature-auth-gcp-workload-identity[REM]
```

Comment:
This is the only one in the table that has a link, just checking that it's okay to have it

Reply:
Yeah, we did this for another z-stream removal recently to make sure it was clear when the removal occurred (since the table is intended more for GA than a z-stream)